process/project-lifecycle.md

```diff
@@ -1,156 +1,168 @@
+* Maintain a diversified contributor base (i.e. not a single-vendor project) with an active flow of contributions
+* Follow security best practices (as recommended by the OpenSSF and others)
+* Maintain a point of contact for vulnerability reports and follow coordinated vulnerability disclosure practices
+* Implement, practice, and refine mature software development and release practices, such as adherence to semantic versioning, and having a declared policy for stable releases and backported fixes.
+
+#### Project Support
+* Receives guidance on technical direction from TAC 
+* Receives space for Contributor Summits at OpenSSF bi-annual events
+* Receives OpenSSF Code of Conduct Committee support
+* Receives infrastructure support (details determined by project leads and OpenSSF Budget Committee)
+* Receives support with vulnerability disclosure from the OpenSSF (Vulnerability Disclosure WG)
+* May post project updates and tutorials to the OpenSSF blog
+* May request OpenSSF budget for project improvements such as security audits or time-bound contracting needs
+* May request OpenSSF for sustained maintainer stipends (details determined by OpenSSF and project leads)
+* With additional TAC or WG approval, may fundraise for dedicated project funds, coordinated by the OpenSSF
+* Projects may use the OpenSSF logo to promote their project (in accordance with the trademark guidelines). Projects may be referred to as an "OpenSSF Project" or "OpenSSF $ProjectName."
+
+#### Graduated Project Entry Requirements and Considerations
+
+* Projects must have maintainers with a minimum of three different company affiliations.
+* Projects must be aligned with the OpenSSF mission and be a novel approach for existing areas or address an unfulfilled need. Projects with duplicate, similar or competing fuctionality to an existing OpenSSF project may be denied Graduation status if the TAC does not see technical justification for overlapping projects. 
+* Projects must be able to show adoption by multiple parties, which could be production deployments or substantial use by established open source communities, and demonstrate the value of that adoption to either the end users or the open source community.
+* Projects must be able to show a consistent release cadence.
+* Projects must have documented project governance and be able to demonstrate that governance in action.
+* When applicable, projects must have completed a security audit through a third party and addressed audit findings and recommendations
 
-#### Gives - Review and Reporting
-*** Need Text ***
+#### Project Graduation Process: Incubating to Graduation
 
-#### Gets - Benefits
-*** Need Text ***
+Graduation requires undergoing due diligence as a part of the process to move from Incubation to Graduation. Due Diligence is driven by a TAC or parent WG sponsor. Once the diligence is complete and the proposal made, two weeks are allocated for public comment before a TAC vote is called.
```

Comment:
```suggestion
Graduation requires undergoing due diligence as a part of the process to move from Incubation to Graduation. Due Diligence is driven by a TAC or parent WG sponsor. For projects seeking Graduation, this may be a light refresh of the existing Due Diligence to cover the additional criteria, or a more in depth Due Diligence depending on the level of change the project has incurred since the original Due Diligence of Incubation was performed. Once the diligence is confirmed by the Sponsor to be complete and the proposal made, the Due Diligence document is made available to the community for two weeks the solicit public comment before a TAC vote is called.
```

Reply:
LGTM